spec/lib/method_hooks_spec.rb

```diff
@@ -5,7 +5,20 @@
   before do
     Object.send(:remove_const, :Base) if Object.const_defined?(:Base)
 
+    module AnotherModuleWIthMethodAddedHook
+      def method_added(method_name)
+        _methods << method_name
+        super
+      end
+
+      def _methods
+        @_methods ||= []
+      end
+    end
+
     class Base
+
+      extend AnotherModuleWIthMethodAddedHook
       extend MethodHooks
```

Comment:
I would move this module definition down to the test that actually needs it (line #97)